src/Hint/Naming.hs

```diff
@@ -49,9 +49,9 @@ namingHint :: DeclHint
 namingHint _ modu = naming $ Set.fromList $ concatMap getNames $ moduleDecls modu
 
 naming :: Set.Set String -> Decl_ -> [Idea]
-naming seen x = [suggestN "Use camelCase" x2 (replaceNames res x2) | not $ null res]
+naming seen x = [suggest "Use camelCase" x x2 [Replace Bind (toSS x) [] (prettyPrint x2)] | not $ null res]
     where res = [(n,y) | n <- nubOrd $ getNames x, Just y <- [suggestName n], not $ y `Set.member` seen]
-          x2 = shorten x
+          x2 = replaceNames res x
```

Comment:
The reason for calling `shorten` here was if you are suggesting to change the name of a function that is 100 lines long then all you see is the body of the function, to `shorten` tries to tidy up the result. Perhaps it should be `shorten $ replaceNames res x`?

Reply:
Sounds good - have shorted `x` as well as `replaceNames res x` in the new commit.